src/main/java/frc/team2412/robot/subsystem/ShooterVisionSubsystem.java

```diff
@@ -11,11 +11,14 @@
 
 public class ShooterVisionSubsystem extends SubsystemBase implements Loggable {
     public static class ShooterVisionConstants {
-        public static final double LIMELIGHT_HEIGHT_OFFSET = 39;
+        public static final double LIMELIGHT_HEIGHT_OFFSET = 37.5;
         public static final double RIM_HEIGHT = 104; // 8ft8in
         public static final double HEIGHT_TO_RIM = RIM_HEIGHT - LIMELIGHT_HEIGHT_OFFSET;
         public static final double HUB_RADIUS = 24;
         public static final double LIMELIGHT_ANGLE_OFFSET = Math.toDegrees(Math.atan2(HEIGHT_TO_RIM, 360 - HUB_RADIUS)); // 10.95
+
+
+        // -0.766666 limelight crosshair offset
```

Comment:
We should consider putting a date stamp in the comment to understand what the latest settings are.